explorer/types.go

```diff
@@ -377,6 +378,7 @@ type Host struct {
 	KnownSince             time.Time `json:"knownSince"`
 	LastScan               time.Time `json:"lastScan"`
 	LastScanSuccessful     bool      `json:"lastScanSuccessful"`
+	LastScanError          string    `json:"lastScanError"`
```

Comment:
Prefer `*string` or `error` so that it's `nil` rather than `""`. The problem with `error` is that you would need a wrapping type for JSON.

Reply:
I went with *string in both.  We return the HostScan as the response for /hosts/%s/scan so we can't use `error` there.